src/openklant/components/klantinteracties/api/serializers/internetaken.py

```diff
@@ -70,7 +85,52 @@ class Meta:
             },
         }
 
+    def to_representation(self, instance):
+        response = super().to_representation(instance)
+        response["toegewezen_aan_actor"] = ActorForeignKeySerializer(
+            instance.actoren.order_by("interneactorenthoughmodel__order").first(),
+            context={**self.context},
+        ).data
+
+        response["toegewezen_aan_actoren"] = ActorForeignKeySerializer(
+            instance.actoren.all().order_by("interneactorenthoughmodel__order"),
+            context={**self.context},
+            many=True,
+        ).data
+        return response
+
+    # TODO: remove when depricated actoren field gets removed
+    def _validate_actoren(self):
+        toegewezen_aan_actor = "toegewezen_aan_actor" in self.initial_data
+        toegewezen_aan_actoren = "toegewezen_aan_actoren" in self.initial_data
+
+        if toegewezen_aan_actor == toegewezen_aan_actoren:
+            if toegewezen_aan_actor:
+                message = _(
+                    "`toegewezen_aan_actor` en `toegewezen_aan_actoren` mag niet beiden gebruikt worden."
+                )
+            else:
+                message = _(
+                    "`toegewezen_aan_actor` of `toegewezen_aan_actoren` is required (mag niet beiden gebruiken)."
+                )
+
+                # If patch don't raise required error
+                if self.context.get("request").method == "PATCH":
+                    return
+
+            raise serializers.ValidationError(message)
+
+    # TODO: remove when depricated actoren field gets removed
+    def _get_actoren(self, actoren):
+        if isinstance(actoren, list):
+            actor_uuids = [str(actor.get("uuid")) for actor in actoren]
+        else:
+            actor_uuids = [glom(actoren, "first.uuid", skip_exc=PathAccessError)]
```

Comment:
I don't quite understand this line. Could you please provide an example when it happens?

Reply:
The structure of both actor fields are different.

The single object returns a validated data of {"first": {"uuid": "..."}}
And the multiple objects return a validated data of [{"uuid": "..."}, {"uuid": "..."}]

This is just a temporary function to fetch the data I need to update and create the objects.